python/python/lance/dataset.py

```diff
@@ -917,16 +919,31 @@ def take_blobs(
 
         Parameters
         ----------
-        row_ids : List Array or array-like
-            row IDs to select in the dataset.
         blob_column : str
             The name of the blob column to select.
+        ids : List Array or array-like
+            row IDs to select in the dataset.
+        addresses: List Array or array-like
+            The (unstable) row addresses to select in the dataset.
+        indices : List Array or array-like
+            The offset / indices of the row in the dataset.
```

Comment:
```suggestion
        ids : Integer Array or array-like
            row IDs to select in the dataset.
        addresses: Integer Array or array-like
            The (unstable) row addresses to select in the dataset.
        indices : Integer Array or array-like
            The offset / indices of the row in the dataset.
```

Technically I think this would be a u64 array.  However, I don't think the method will work if given any kind of pyarrow array?